frontend/src/app/app.component.spec.ts

```diff
@@ -62,8 +43,10 @@ describe('AppComponent', () => {
         NoopAnimationsModule,
         CommonModule
       ],
-      providers: [{ provide: OAuthService,
-        useValue: oauthServiceMock }],
+      providers: [{
+        provide: OAuthService,
+        useValue: oauthServiceMock
```

Comment:
why is oauth service mock used here but deleted above?

Reply:
Only the content of the service was deleted because it was never used the mockservice still exists